pkg/utils/httpserver/server.go

```diff
@@ -4,13 +4,20 @@ import (
 	"context"
 	"fmt"
 	"net/http"
+	"strings"
 
 	"go.uber.org/zap"
 )
 
 func StartServer(ctx context.Context, log *zap.Logger, svc string, port string, handler http.Handler) {
+	var addr string
+	if strings.Contains(port, ":") {
```

Comment:
if !strings.Contains(port, ":") {
      port = fmt.Sprintf(":%s", port)
)

can be done in single condition.